traveltimepy/dto/common.py

```diff
@@ -107,6 +107,10 @@ class Property(str, Enum):
     FARES = "fares"
 
 
+class PropertyProto(int, Enum):
```

Comment:
you can add ferries too

Reply:
I thought adding fares would be misleading for the user since we don't support it here for now. I can add it tho